spec/hyperYieldSpec.js

```diff
@@ -0,0 +1,20 @@
+describe('hyper-yield tag', function() {
+  var $compile,
+      $rootScope;
+
+  beforeEach(module('hyperContentFor'));
+
+  beforeEach(inject(function(_$compile_, _$rootScope_){
+    $compile = _$compile_;
+    $rootScope = _$rootScope_;
+  }));
+
+  it('Replace with content from matching hyper-content', function() {
```

Comment:
I'd actually like this to read like this:

```javascript
it("is replaced with ...");
```

Seeing as “it” is the tag itself, that is probably the most correct, English-wise. :smiley: